cmd/gangway/handlers.go

```diff
@@ -89,17 +132,19 @@ func generateKubeConfig(tmplFile string, data interface{}) {
 	// create buffered io writer
 	w := bufio.NewWriter(f)
 
-	tmpl := template.New(tmplFile).Funcs(FuncMap())
-	tmpl, err = tmpl.Parse(string(templateData))
+	d, err := yaml.Marshal(&kcfg)
+
 	if err != nil {
-		log.Errorf("Error parsing kubeconfig template: %s", err)
+		log.Fatalf("error: %v", err)
 	}
-	err = tmpl.ExecuteTemplate(w, tmplFile, data)
+
+	w.Write(d)
```

Comment:
Looks like we are still writing to disk?

Reply:
oh hrmmm, I had a commit that cleaned this out that apparently I never pushed up.  I'm digging for it, I may need to rewrite it, but shouldn't be too hard.